twistedchecker/checkers/python3.py

```diff
@@ -1,187 +0,0 @@
-"""
-Checker for python3 compatibility issues.
```

Comment:
Although we don't have the ``except Exception, e`` check. Would be really cool if this would be included upstream. ;-)